lib/teamcity/oracle/oracle_installation.txt

```diff
@@ -1,12 +1,15 @@
 
-Installation steps for Oracle (Express Edition) for NH TeamCity:
+Installation steps for Oracle for NH TeamCity:
 
-1.  Download Oracle Database 10g Express Edition (OracleXE.exe):  http://www.oracle.com/technetwork/database/express-edition/downloads/102xewinsoft-090667.html;
+1.  Download Oracle Database 12c (12.2):  http://www.oracle.com/technetwork/database/enterprise-edition/downloads/index.html
```

Comment:
12.2 does not work:
1. 12.2 ManagedDataAccess fails with this stupid bind by name + comments bug (about 8 tests are failing).
2. 12.1 (the one which does not fail tests) ManagedDataAccess driver does not work with 12.2 because of "protocol violations".